criu/proc_parse.c

```diff
@@ -105,6 +106,18 @@ bool is_vma_range_fmt(char *line)
 	return __is_vma_range_fmt(line);
 }
 
+bool handle_vma_plugin(int *fd, struct stat *stat)
+{
+	int ret;
+	ret = run_plugins(HANDLE_DEVICE_VMA, *fd, stat);
```

Comment:
I think we usually have an empty line between declaration and code.

Reply:
ACK